src/mono/mono/mini/driver.c

```diff
@@ -216,6 +216,11 @@ parse_debug_options (const char* p)
 		} else if (!strncmp (p, "gdb", 3)) {
 			opt->gdb = TRUE;
 			p += 3;
+#ifdef ENABLE_NETCORE
+		} else if (!strncmp (p, "ignore", 6)) {
+			opt->enabled = FALSE;
```

Comment:
what is setting `opt->enabled = TRUE`?

Reply:
I set it to true at the start of the function